SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -170,7 +170,7 @@ public static dbPhoneNumber ToEntity(this PhoneNumber number, long personId, str
         {
             return new dbPhoneNumber
             {
-                Number = number.Number.ToString(),
+                Number = number.Number,
```

Comment:
Why was this change made? 🤔

Reply:
It was the last warning I was getting in the file and I wanted that Rider green tick.

The type of number.Number is already a string (even if the naming of number.Number might suggest otherwise), calling .ToString() against a string just returns itself so it could be removed.

Reply:
Wicked, makes sense! ✅